WIP: Update classes (#388)
EMMC-ASBL/tripper: docs/datadoc/classdoc.md

Documenting abstract resources
------------------------------

> Resources that doesn't exists are called *abstract resources*.
> A typical example of an abstract resource is a procedure that haven't been performed (yet), like a general description of a task in a workflow.
> 

Reviewer: ```suggestion
Resources that do not exist are called *abstract resources*.
A typical example of an abstract resource is a procedure that has not been performed (yet), like a general description of a task in a workflow.

```
